scripts/generate-k8s-yaml

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -euo pipefail
+
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+
+PLATFORM=$(uname | tr '[:upper:]' '[:lower:]')
+HELM_VERSION="3.0.2"
+
+MAKEFILEPATH=$SCRIPTPATH/../Makefile
+VERSION=$(make -s -f $MAKEFILEPATH version)
+BUILD_DIR=$SCRIPTPATH/../build/k8s-resources
+INDV_RESOURCES_DIR=$BUILD_DIR/indv-resources
+TAR_RESOURCES_FILE=$BUILD_DIR/aws-node-termination-handler-k8s-resources-$VERSION.tar
+AGG_RESOURCES_YAML=$BUILD_DIR/aws-node-termination-handler-$VERSION.yaml
+mkdir -p $BUILD_DIR/indv-resources
+
+curl -L https://get.helm.sh/helm-v$HELM_VERSION-$PLATFORM-amd64.tar.gz | tar zxf - -C $BUILD_DIR
+mv $BUILD_DIR/$PLATFORM-amd64/helm $BUILD_DIR/.
+rm -rf $BUILD_DIR/$PLATFORM-amd64
+chmod +x $BUILD_DIR/helm
+
+$BUILD_DIR/helm template aws-node-termination-handler \
+    --namespace kube-system \
```

Comment:
nit: might be worth parameterizing the namespace instead of hard-coding it to kube-system?